libmachine/provision/configure_swarm.go

```diff
@@ -50,6 +50,9 @@ func configureSwarm(p Provisioner, swarmOptions swarm.Options, authOptions auth.
 			swarmOptions.Strategy,
 			advertiseMasterInfo,
 		)
+		if swarmOptions.IsExperimental {
```

Comment:
OK, weird, in other places in the docs we use double dash, e.g. https://github.com/docker/swarm/blob/master/docs/discovery.md. @abronan @vieux @aluzzardi do you all have a preference on this?  `--experimental` will work as well as `-experimental` right?